x-pack/plugins/lens/public/indexpattern_plugin/dimension_panel.test.tsx

```diff
@@ -0,0 +1,404 @@
+/*
```

Comment:
We are testing whether the helpers `getColumnOrder` and `getPotentialColums` work correctly in isolation, but not whether they are used correctly within the dimension panel itself. In addition to the direct unit tests I think a few higher level test are useful, e.g. whether the `suggestedOrder` prop is correctly used when set on a dimensional panel or whether only the "right" fields are shown depending on the operations filter

Reply:
Take another look, I think I've tested this more completely in the latest. Make sure to open up the new `__mocks__/operations.ts` file

Reply:
LGTM - I think a higher level functional test as discussed offline can be added in a separate PR